src/topics/unread.js

```diff
@@ -93,15 +93,17 @@ module.exports = function (Topics) {
 
 		params.cutoff = await Topics.unreadCutoff(params.uid);
 
-		const [followedTids, ignoredTids, categoryTids, userScores, tids_unread] = await Promise.all([
+		const [followedTids, ignoredTids, categoryTids, userScores, tids_unread, postCounts] = await Promise.all([
 			getFollowedTids(params),
 			user.getIgnoredTids(params.uid, 0, -1),
 			getCategoryTids(params),
 			db.getSortedSetRevRangeByScoreWithScores(`uid:${params.uid}:tids_read`, 0, -1, '+inf', params.cutoff),
 			db.getSortedSetRevRangeWithScores(`uid:${params.uid}:tids_unread`, 0, -1),
+			db.getSortedSetRevRangeWithScores(`topics:posts`, 0, -1),
```

Comment:
Need to remove this since it loads all topics ever created.

Reply:
:+1: Completely removed.